cosmo_tester/test_suites/broker_management/manager_test.py

```diff
@@ -0,0 +1,79 @@
+import json
+
+
+def test_broker_management(brokers_and_manager, logger):
+    # All in one test for speed until such time as complexity of these
+    # operations increases to the point that extra tests are needed.
+    broker1, broker2, manager = brokers_and_manager
+    manager.put_remote_file(
+        local_path=broker1.ca_path,
+        remote_path='/tmp/rabbit.ca',
+    )
+
+    manager.additional_install_config = {
+        'rabbitmq': {
+            'ca_path': '/tmp/rabbit.ca',
+            'cluster_members': {
+                broker1.hostname: {
+                    'default': str(broker1.private_ip_address),
+                }
+            }
+        },
+        'services_to_install': ['database_service', 'manager_service'],
+    }
+    manager.bootstrap()
+
+    manager.enter_sanity_mode()
+
+    expected_1 = {
+        'port': 5671,
+        'networks': {'default': str(broker1.private_ip_address)},
+        'name': broker1.hostname,
+    }
+    expected_2 = {
+        'port': 5671,
+        'networks': {'default': str(broker2.private_ip_address),
+                     'testnet': '192.0.2.4'},
+        'name': broker2.hostname,
+    }
+
+    logger.info('Confirming list functionality.')
+    brokers_list = list_brokers(manager)
+    assert len(brokers_list) == 1
+    assert expected_1 in brokers_list
```

Comment:
those two asserts are just `assert brokers_list == [expected_1]`. One line saved! (and same for the post-remove-check for two lines saved!)